snapcraft/tests/unit/test_lxd.py

```diff
@@ -606,6 +606,19 @@ def call_effect(*args, **kwargs):
                 ContainerConnectionError,
                 self.make_containerbuild().execute)))
 
+    def test_debug_messages(self):
+        class TestFormatter(logging.Formatter):
+            def format(self, record):
+                message = super().format(record)
+                return '[{}]{}'.format(record.levelname, message)
+
+        self.fake_logger = fixtures.FakeLogger(
+            format='%(message)s', formatter=TestFormatter, level=logging.DEBUG)
+        self.useFixture(self.fake_logger)
+
+        self.make_containerbuild().execute()
+        self.assertIn('[DEBUG]Terminating', self.fake_logger.output)
```

Comment:
I like the testtools version better: `self.assertThat(self.fake_logger.output, Contains('[DEBUG]Terminating'))`.

Reply:
Done